src/controller/CHIPDeviceControllerSystemState.h

```diff
@@ -150,6 +173,10 @@ class DeviceControllerSystemState
     secure_channel::MessageCounterManager * mMessageCounterManager = nullptr;
     FabricTable * mFabrics                                         = nullptr;
     CASEServer * mCASEServer                                       = nullptr;
+    CASESessionManager * mCASESessionManager                       = nullptr;
+    SessionIDAllocator * mSessionIDAllocator                       = nullptr;
+    OperationalDevicePool * mOperationalDevicePool                 = nullptr;
+    CASEClientPool * mCASEClientPool                               = nullptr;
```

Comment:
Likewise with these entities.  Can this code accommodate having zero-size for these pools? We don't use them.

Reply:
If the pool code compiles with the pool size set to 0, and the CASESessionManager is never used, then these things would never get touched either.